settings.gradle

```diff
@@ -9,7 +9,7 @@ plugins {
     id 'io.micronaut.build.shared.settings' version '7.3.1'
 }
 
-rootProject.name = 'project-template-parent'
+rootProject.name = 'micronaut-build-plugin-sourcegen'
```

Comment:
This should probably be `micronaut-build-plugin-sourcegen-parent` to be consistent with other projects.